modules/docs/arrow-docs/docs/docs/integrations/retrofit/README.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: docs
+title: Retrofit
+permalink: /docs/integrations/retrofit/
+---
+
+## Retrofit
+
+{:.advanced}
+advanced
+
+Arrow contains a integration module for Retrofit so you can have a Arrow integration in your endpoints definition.
+
+Define your endpoints and define for CallK as the return Type:
+
+```kotlin
+interface ApiClientTest {
+
+  @GET("test")
+  fun testCallK(): CallK<ResponseMock>
+
+  @GET("testCallKResponse")
+  fun testCallKResponse(): CallK<ResponseMock>
+
+  @POST("testResponsePOST")
+  fun testIOResponsePost(): CallK<Unit>
+
+}
+```
+
+You can use CallK to have Async/MonadDefer/MonadError intances as your data wrapper.
```

Comment:
`Async`, `MonadDefer`, `MonadError`, and make them link to their respective sites. You need to add a relative link, like the ones at the bottom.